### PR TITLE
Api server/seed

### PR DESCRIPTION
**Pull request by huynonstop**
Add seed for customer, identity, account
Add dictionary for status and type account

### CODE REVIEW

File: packages/api-server/utils/statusDict.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const STATUS = {
```
Comment (kerbal) [inline packages/api-server/utils/statusDict.js:1]: Create a new folder `constants` and put these constants in

File: packages/api-server/utils/typeAccountDict.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const TYPE = {
```
Comment (kerbal) [inline packages/api-server/utils/typeAccountDict.js:1]: ACCOUNT_TYPE

File: packages/api-server/utils/statusDict.js (hunk 1 above)
Comment (kerbal) [inline packages/api-server/utils/statusDict.js:1]: ACCOUNT_STATUS

File: packages/api-server/utils/typeAccountDict.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  DEFAULT: 'DEFAULT',
+  CHECKING: 'CHECKING',
```
Comment (kerbal) [inline packages/api-server/utils/typeAccountDict.js:3]: CREDIT :thinking:

File: packages/api-server/utils/statusDict.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+  UNVERIFIED: 'UNVERIFIED',
+  VERIFIED: 'VERIFIED',
+  NORMAL: 'NORMAL',
```
Comment (kerbal) [inline packages/api-server/utils/statusDict.js:4]: what does it mean?

File: packages/api-server/utils/typeAccountDict.js (hunk 3 above)
Comment (huynonstop) [reply packages/api-server/utils/typeAccountDict.js:3]: https://sotaydoanhtri.com/thuat-ngu/credit-account-11662
https://en.wikipedia.org/wiki/Transaction_account

File: packages/api-server/utils/statusDict.js (hunk 4 above)
Comment (huynonstop) [reply packages/api-server/utils/statusDict.js]: UNVERIFIED and VERIFIED is the status for customers, NORMAL is the status for accounts. Those status constants will be more clear in the next commit

**Pull request merged**